test/semaphore.js

```diff
@@ -8,36 +8,46 @@ const {
   SemaphoreDegrees,
   SemaphoreDirection,
   SemaphoreEncoding,
-  SemaphoreStream
+  SemaphoreStream,
 } = require('../');
 
 // eslint:disable:
 
-describe('Semaphore', function () {
-  describe('Characters', function () {
-    it('Char Matches', function () {
-      const aChar = new SemaphoreCharacter(SemaphoreDirection.SouthWest | SemaphoreDirection.South);
+describe('Semaphore', () => {
+  describe('Characters', () => {
+    it('Char Matches', () => {
+      const aChar = new SemaphoreCharacter(
+        SemaphoreDirection.SouthWest | SemaphoreDirection.South
+      );
       assert.strictEqual(aChar.toString(), 'A/1');
 
       const iChar = new SemaphoreCharacter(SemaphoreEncoding.Number9);
       assert.strictEqual(iChar.toString(), 'I/9');
 
-      const qChar = new SemaphoreCharacter(SemaphoreDirection.West | SemaphoreDirection.NorthEast);
+      const qChar = new SemaphoreCharacter(
+        SemaphoreDirection.West | SemaphoreDirection.NorthEast
+      );
       assert.strictEqual(qChar.toString(), 'Q');
 
-      const zChar = new SemaphoreCharacter(SemaphoreDirection.SouthEast | SemaphoreDirection.East);
+      const zChar = new SemaphoreCharacter(
+        SemaphoreDirection.SouthEast | SemaphoreDirection.East
+      );
       assert.strictEqual(zChar.toString(), 'Z');
     });
 
-    it('No Matches', function () {
-      const noChar = new SemaphoreCharacter(SemaphoreDirection.South | SemaphoreDirection.South);
+    it('No Matches', () => {
+      const noChar = new SemaphoreCharacter(
+        SemaphoreDirection.South | SemaphoreDirection.South
+      );
       assert.strictEqual(noChar.toString(), '');
 
-      const emptyChar = new SemaphoreCharacter(SemaphoreDirection.South | SemaphoreDirection.South);
+      const emptyChar = new SemaphoreCharacter(
+        SemaphoreDirection.South | SemaphoreDirection.South
+      );
```

Comment:
Isn't this exactly the same test as the one above it?

Reply:
Removed it, looks like it has been there since the initial commit.